windrecorder/ui/state.py

```diff
@@ -46,9 +55,9 @@ def render():
         with col2a:
             st.session_state.Stat_query_Month = st.number_input(
                 label="Stat_query_Month",
-                min_value=selector_month_min,
-                max_value=selector_month_max,
-                value=st.session_state.stat_db_latest_datetime.month,
+                min_value=get_year_db_earliest_month_and_latest_month(st.session_state.Stat_query_Year)[0],
+                max_value=get_year_db_earliest_month_and_latest_month(st.session_state.Stat_query_Year)[1],
+                value=get_year_db_earliest_month_and_latest_month(st.session_state.Stat_query_Year)[1],
```

Comment:
Please keep [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). Save `get_year_db_earliest_month_and_latest_month(st.session_state.Stat_query_Year)` to a variable and use it. And after that, there's no need to create a local function which makes the code harder to read.

Reply:
thanks for your knowledge! i make it wetter and pls have another look <3